dojo/settings/settings.dist.py

```diff
@@ -801,8 +801,8 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
     'Acunetix Scan': ['title', 'description'],
     'Trivy Scan': ['title', 'severity', 'cve', 'cwe'],
     'Snyk Scan': ['vuln_id_from_tool', 'file_path', 'component_name', 'component_version'],
+    'GitLab Dependency Scanning Report': ['title', 'cwe', 'cve', 'file_path', 'component_name', 'component_version', 'severity'],
```

Comment:
do we really need severity to be part of the hashcode calculation? always has felt weird to me. if there are two issues with all fields the same, except severity, aren't they just the same vulnerability but just the severity was changed?

Reply:
Agree, your thought makes sense.
I also removed CWE, since it's not actually filled by the GitLab dependency scanner.